e2e/tests/ui-driven/src/pages/Editor.ts

```diff
@@ -34,7 +34,6 @@ import { selectedFlag } from "../helpers/globalHelpers.js";
 
 export class PlaywrightEditor {
   readonly page: Page;
-  readonly addNewServiceButton: Locator;
```

Comment:
This only had a single use case, I've removed it as a property and I'm just calling `this.page` instead.